api/envoy/extensions/common/ratelimit/v3/ratelimit.proto

```diff
@@ -130,3 +130,15 @@ message LocalRateLimitDescriptor {
   // Token Bucket algorithm for local ratelimiting.
   type.v3.TokenBucket token_bucket = 2 [(validate.rules).message = {required: true}];
 }
+
+// Configuration that used to enable local cluster level rate limiting where the token buckets
```

Comment:
nit: /s/"Configuration that used to enable local"/"Configuration used to enable local"